user/travis-pro.md

```diff
@@ -61,3 +61,16 @@ Travis CI is, and always will be, free for open source projects.
 
 For a list of plans and prices for private repositories, look at
 [travis-ci.com/plans](https://travis-ci.com/plans).
+
+## Making a private repository public
+
+If you have a private repository that you'd like to make public, first change
+the setting on GitHub, then resync your Travis CI account:
+
+1. Follow the [instructions on
+GitHub](https://help.github.com/articles/making-a-private-repository-public/) on
```

Comment:
From our experience, we normally fix this by de-activating the hook for private repos and activating it on public ones (or vice-versa). Could we adapt the instructions to match this? Thank you!

The notify URL is currently a way for us to understand what happens on the inside, besides from the hook being different. I think it's alright to leave it out, to avoid confusion and people manually updating URLs - but if we find that it's helpful support-wise we can add it later on. What do you think?